### PR TITLE
RRULE support in downtimes

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Add details about RRULE support in downtimes

### Motivation
This was already supported in the API but the UI has been added for it.

### Preview

- https://docs-staging.datadoghq.com/nils/downtimes-rrules/monitors/downtimes/?tab=onetime#schedule-1
- https://docs-staging.datadoghq.com/nils/downtimes-rrules/monitors/guide/supress-alert-with-downtimes/?tab=api#recurring-downtime-on-the-nth-weekday-of-the-month
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/downtimes.md [hunk 1]
```diff
@@ -56,6 +58,19 @@ Set a recurring downtime by entering the start date, time, time zone, repeat, an
 
 When a single downtime of a recurring downtime ends, the single downtime is cancelled and a new downtime is created with the same constraints and updated start and end times. **Note**: The original creator is associated to all the newly created downtimes.
 
+{{< img src="monitors/downtimes/downtime_recurring.jpg" alt="downtime on alert"  style="width:80%;">}}
+
+RRULEs - or [recurrence rules][1] - are also available to define Downtimes schedule. [This official RRULE generator][2] might be helpful to generate such recurring rules.
```
Comment (apigirl) [inline content/en/monitors/downtimes.md:63]:
```suggestion
Use RRULEs - or [recurrence rules][1] - to define downtimes schedules. Use the official [RRULE generator][2] as a tool to generate recurring rules.
```

File: content/en/monitors/downtimes.md [hunk 2]
```diff
@@ -56,6 +58,19 @@ Set a recurring downtime by entering the start date, time, time zone, repeat, an
...
+
+Use them to define downtimes on specific days of the month. For instance, on the third Monday of each month:
```
Comment (apigirl) [inline content/en/monitors/downtimes.md:65]:
```suggestion
A common use case is to use RRULES to define downtimes on specific days of the month. For example, on the third Monday of each month:
```

File: content/en/monitors/downtimes.md [hunk 3]
```diff
@@ -56,6 +58,19 @@ Set a recurring downtime by entering the start date, time, time zone, repeat, an
...
+
+{{< img src="monitors/downtimes/downtime_rrule.jpg" alt="downtime on alert"  style="width:80%;">}}
+
+**Note**: Attributes specifying the duration in RRULE are not supported (for example, `DTSTART`, `DTEND`, `DURATION`), see the RFC for the possible attributes.
```
Comment (apigirl) [inline content/en/monitors/downtimes.md:69]: is there a link to the RFC?

**Review by apigirl — COMMENTED** [review]
some suggestions to clarify phrasing and one question about adding a link

File: content/en/monitors/downtimes.md (hunk 3 above)
Comment (NBParis) [reply content/en/monitors/downtimes.md:69]: I added a link when I first introduced recurrence rules, do you think we should add another one here?

Comment (NBParis) [reply content/en/monitors/downtimes.md:69]: But good point, we should likely remove the mention to the RFC or add a link ;) on it

**Review by apigirl — APPROVED** [review]

**Pull request merged**